e2e/test/scenarios/question/questions-entity-id.cy.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { H } from "e2e/support";
```

Comment:
I didn't found a generic questions e2e file, so I created a new one, I'm unsure if it's ok as I think there's cypress takes some time to open files